daemon/test/test.sh

```diff
@@ -5,7 +5,13 @@ cd `git rev-parse --show-toplevel`/daemon/test
 
 . scripts/vars.sh
 
-scripts/setup.sh
+# If bitcoind not already running, start it.
+if ! $CLI getinfo >/dev/null 2>&1; then
```

Comment:
No, see below: if we didn't start it, SHUTDOWN_BITCOIN is simply /bin/true. which does nothing.